build_runner_core/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.2.0+1
```

Comment:
synced offline about making this `0.2.1` and pushing a  new version of `build_runner` as well. Fine to merge on red since we'll have a dep on an unpublished package.